smac/scenario/scenario.py

```diff
@@ -345,6 +359,40 @@ def extract_instance_specific(instance_list):
         else:
             self.logger.info("Output to %s" % (self.output_dir))
 
+    def _write(self):
+        """Write scenario to file in a format that can be easily retrieved using
+        the input_reader. Will overwrite if file already exists.
+
+        Sideeffect: creates output-directory if it doesn't exist.
+        """
+        self.logger.debug("Writing scenario-file to {}.".format(self.output_dir))
+        # Reverse mapping of option-names to internal -> external
+        options_int2ext = {v: k for k, v in self.options_ext2int.items()}
+
+        # We need to modify certain values when writing them
+        def modify(key, value):
+            ''' returns the correct value to write into a file '''
+            if key == 'ta' : return " ".join(value)
+            else: return value
+
+        # Create output-dir if necessary
+        if not os.path.isdir(self.output_dir):
+            self.logger.debug("Output directory does not exist! Will be created.")
+            try:
+                os.makedirs(self.output_dir)
+            except OSError:
```

Comment:
This would fail if the output directory already exists. Is this intended behaviours?